### ISSUE TITLE
Exception when importing Netflix date

### ISSUE THREAD

**Issue by JVT038** [leepeuker/movary#260]
While testing, I instantly encountered this error during import:
```
[2023-01-29T18:03:38.408116+01:00] movary.WARNING: Netflix: Could not parse activity csv {"exception":"[object] (Exception(code: 0): Failed to parse time string (31/12/2022) at position 0 (3): Unexpected character at C:\\xampp\\htdocs\\Projecten\\movary\\src\\ValueObject\\Date.php:28)
[stacktrace]
#0 C:\\xampp\\htdocs\\Projecten\\movary\\src\\ValueObject\\Date.php(28): DateTime->__construct('31/12/2022')
#1 C:\\xampp\\htdocs\\Projecten\\movary\\src\\Service\\Netflix\\NetflixActivityCsvParser.php(32): Movary\\ValueObject\\Date::createFromString('31/12/2022')
#2 C:\\xampp\\htdocs\\Projecten\\movary\\src\\HttpController\\NetflixController.php(64): Movary\\Service\\Netflix\\NetflixActivityCsvParser->parseNetflixActivityCsv('C:\\\\xampp\\\\tmp\\\\ph...')
#3 [internal function]: Movary\\HttpController\\NetflixController->processNetflixActivity(Object(Movary\\ValueObject\\Http\\Request))
#4 C:\\xampp\\htdocs\\Projecten\\movary\\vendor\\php-di\\invoker\\src\\Invoker.php(74): call_user_func_array(Array, Array)
#5 C:\\xampp\\htdocs\\Projecten\\movary\\vendor\\php-di\\php-di\\src\\Container.php(248): Invoker\\Invoker->call(Array, Array)
#6 C:\\xampp\\htdocs\\Projecten\\movary\\public\\index.php(38): DI\\Container->call(Array, Array)
#7 {main}
"} []
```
For some reason the string`31/12/2022` causes a crash and I can't figure out why.
This individual test also shows the same error.
![image](https://user-images.githubusercontent.com/47184046/215343318-b3c2a3e5-bbc6-4801-ae9a-ada25d25f164.png)

**Comment by JVT038** [comment]
When changing it to `$dateObject = \DateTime::createFromFormat('dd/mm/YY', $date);` it returns false.

**Comment by JVT038** [comment]
`$dateObject = \DateTime::createFromFormat('d/m/Y', $date);` does work though.
So to prevent this issue, you'll probably need to fix #259 and either allow a custom format entered by the user or have multiple, hardcoded formats being tried until it works.
I'll probably close this in favour of #259
